dev_tools/utils/nxdl_utils.py

```diff
@@ -246,10 +248,7 @@ def belongs_to_capital(params):
             ):
                 continue
             # check if the name of another sibling fits better
-            name_any2 = (
-                "nameType" in child2.attrib.keys()
-                and child2.attrib["nameType"] == "any"
-            )
+            name_any2 = child2.attrib.get("nameType") == "any"
```

Comment:
LHS and RHS different LHS is tuple while RHS evaluates on the right-hand side of the expression a boolean but not a boolean in a tuple